### PR TITLE
build: Cancel concurrent PR jobs

### PR DESCRIPTION
**Pull request by gastaldi**
GitHub will automatically cancel concurrent jobs per PRs

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -6,6 +6,10 @@ on:
 permissions:
   contents: read
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}-${{ matrix.os }}
```
Comment (aalmiray) [inline .github/workflows/build.yml:10]: Is `matrix.os` resolvable here? Or should it be the runner's OS?

Comment (gastaldi) [reply .github/workflows/build.yml:10]: https://docs.github.com/en/enterprise-cloud@latest/actions/writing-workflows/choosing-what-your-workflow-does/control-the-concurrency-of-workflows-and-jobs says it is

**Comment by aalmiray** [comment]
Thanks

**Comment by aalmiray** [comment]
@gastaldi as I thought, the `matrix` element is not implicitly defined https://github.com/jreleaser/jreleaser/actions/runs/10971972024

This means PR's won't run at all 😢

**Comment by gastaldi** [comment]
Great, so the docs is misleading in saying that `matrix` is allowed. Anyway, it should work if you remove just that expression, let me submit a PR

**Pull request merged**